_includes/news/release_summary.html

```diff
@@ -32,6 +32,7 @@
         <a href="https://www.npmjs.com/package/clarity-angular"
            target="_blank">clarity-angular</a>
         {% if include.hasIcons %}
+           <br>
```

Comment:
This might not work on Firefox. I remember using `br` for separating headers in the headers demo and it didn't work in Firefox. Might have to use margin